specs/fault-proof/stage-one/anchor-state-registry.md

```diff
@@ -102,39 +111,49 @@ system's `DisputeGameFactory` contract.
+
+### Retirement Timestamp
+
+The **Retirement Timestamp** is a timestamp value maintained within the `AnchorStateRegistry` that
+can be used to invalidate games. Games with a creation timestamp less than or equal to the
+Retirement Timestamp are automatically considered to be invalid.
+
+The RetirementTimestamp has the effect of retiring all games created before the specific
+transaction in which the retirement timestamp was set. This includes all games created in the same
+block as the transaction that set the Retirement Timestamp. We acknowledge the edge-case that games
+created in the same block *after* the Retirement Timestamp was set will be considered Retired Games
+even though they were technically created "after" the Retirement Timestamp was set.
 
 ### Retired Game
 
 A Dispute Game is considered to be a **Retired Game** if the game contract was created with a
-timestamp less than or equal to the retirement timestamp (`respectedGameTypeUpdatedAt`) defined in
-the `OptimismPortal` contract. The retirement timestamp has the effect of retiring all games
-created before the specific transaction in which the retirement timestamp was set. This includes
-all games created in the same block as the transaction that set the retirement timestamp. We
-acknowledge the edge-case that games created in the same block *after* the retirement timestamp
-was set will be considered Retired Games even though they were technically created after the
-retirement timestamp was set.
+timestamp less than or equal to the [Retirement Timestamp](#retirement-timestamp).
 
 ### Proper Game
 
 A Dispute Game is considered to be a **Proper Game** if it has not been invalidated through any of
-the mechanisms defined by the `OptimismPortal` contract. A Proper Game is, in a sense, a "clean"
-game that exists in the set of games that are playing out correctly in a bug-free manner. A Dispute
-Game can be a Proper Game even if it has not yet resolved or resolves in favor of the Challenger.
-A Game that was previously a Proper Game can no longer be a Proper Game if it is later invalidated.
+the mechanisms defined by the `AnchorStateRegistry` contract. A Proper Game is, in a sense, a
+"clean" game that exists in the set of games that are playing out correctly in a bug-free manner. A
+Dispute Game can be a Proper Game even if it has not yet resolved or resolves in favor of the
+Challenger. A Game that was previously a Proper Game can no longer be a Proper Game if it is later
+invalidated.
 
 Specifically, a game is considered to be a Proper Game if all of the following are true:
 
-- The game is a Registered Game
-- The game is not a Blacklisted Game
-- The game is not a Retired Game
+- The game is a [Registered Game](#registered-game)
+- The game is **NOT** a [Blacklisted Game](#blacklisted-game)
+- The game is **NOT** a [Retired Game](#retired-game)
+
+Additionally, ALL games will NOT be considered to be Proper Games while the Superchain-wide pause
+is active.
```

Comment:
While not a problem, really important to point out in the incident response runbooks that un-pausing after a few days of being paused, without also updating the retirement timestamp, could very quickly get the system into a dangerous state.

Reply:
This also kind-of conflicts with "A Game that was previously a Proper Game can no longer be a Proper Game if it is later invalidated," though might be mincing words there. From this, it sounds like games can transition from `proper -> improper -> proper`, if the only thing that made a game temporarily improper was that the superchain was paused.

Reply:
Also followed up on here: https://github.com/ethereum-optimism/specs/pull/569/files#diff-7565371fb91f003706461dfdebc80ed23edc8e2d89b4bfeda5c41f22b58d5e2cR263-R264

Reply:
Great points, will make sure this is very clearly called out

Reply:
Changes look good here.